index/version.txt

```diff
@@ -1 +1 @@
-0.3.1
+0.3.2
```

Comment:
@alexgleith I was just going to start the process of upgrading to 0.3.1 but noticed this. Should there be a release each time the version is bumped?

Reply:
Oh hey, sorry, I forgot you did that work on the release process...

I think if we're using releases to do the versioning, we should remove this file and make sure the github action grabs the version from the git tag.

Like how [this works](https://github.com/digitalearthpacific/dep-ml-products/blob/main/.github/workflows/build-test-docker.yml#L75).

``` python
      - name: Get version number for tagging the image
        id: get_version
        run: |
          git fetch --all --tags
          echo "VERSION=$(git describe --tags)" >> $GITHUB_OUTPUT
```

Reply:
Well, don't pay too much attention to me, I'm still trying to dig myself out of the hole I'm in..

I don't need Azure support, so next week I'll upgrade to the latest explorer/ows, and this to 0.3.1 since that contains a lot of the improvements I want. I've put a link to your comment in the issue so I can find it when I manage to get things back on track again.